doc/Learn/Tutorials/Navigation/Advanced/HowTo-UseNavigationView.md

```diff
@@ -163,14 +163,102 @@ Choosing the right control for your navigation needs is important, and one commo
   </Grid>
   ```
 
-* Observe how the `NavigationView` and the content area are now connected. When you select a `NavigationViewItem`, the corresponding `Grid` will be shown.
-
 * Set the `Visibility` of the `Grid` elements to `Collapsed` to hide the content area's children beforehand:
 
   ```xml
   Visibility="Collapsed"
   ```
 
+#### Navigating to Page elements
+
+* You may want to navigate to a `Page` view element represented by a route name. It is possible to do this without defining the view element alongside the other content areas. For instance, you may need to display a login page `LoginPage` which will be defined in a separate XAML file.
```

Comment:
I'm not sure having a loginpage within a tabbed view makes sense. Can we change this to perhaps a products page. That way we can also show what happens when they click on a product - it navgiates to the details in the tab